slu-scripts/src/index.ts

```diff
@@ -46,6 +47,7 @@ export async function execScript(argv: any) {
 	const encryptedDataPath: string | undefined = process.env.npm_config_input_enc;
 	const registrationUrl: string | undefined = process.env.npm_config_registration_url;
 	const isConfigFile: string | undefined = process.env.npm_config_is_config_file;
+	const collectorDataUrl: string | undefined = process.env.npm_config_collector_data_url;
```

Comment:
We now have two env vars for the collector url
```
process.env.npm_config_collector_url = 'http://localhost:4000/collector/';
process.env.npm_config_collector_data_url = 'http://localhost:3000/collector/data';
```

It would be better to have the base url as collector_base_url and then adjust the paths based on what will be queried